tools/scripts/rebuildSoundLibraryManifest.rb

```diff
@@ -278,6 +285,16 @@ def build_sound_metadata(sound_objects, previous_metadata)
       begin
         json_response = objects['json'].get
         metadata = JSON.parse(json_response.body.read)
+        aliases = metadata['aliases']
+        # move categories out of aliases
+        categories = []
+        aliases.each do |a|
+          if a.start_with? "category_"
+            categories.push (a.delete_prefix "category_")
+          end
+        end
+        metadata['aliases'] = aliases.reject {|a| a.start_with? "category_"}
```

Comment:
don't we want to keep the categories in our list of aliases (minus the "category_" prefix)?

e.g., if the search query is "background," we would want to return everything in the background category. right now, only navigating to the background category via the UI would give those results, right?